tests/ui.rs

```diff
@@ -312,21 +440,30 @@ fn main() -> Result<()> {
         }
     }
 
-    ui(Mode::Pass, "tests/pass", &target, WithoutDependencies, tmpdir.path())?;
-    ui(Mode::Pass, "tests/pass-dep", &target, WithDependencies, tmpdir.path())?;
+    ui(Mode::Pass { rustc: false }, "tests/pass", &target, WithoutDependencies, tmpdir.path())?;
```

Comment:
Can you also adjust the `eprintln` in `fn ui` to add something like "on rustc" or so, so that one can see which way the tests are being run?